oqsprov/oqs_sig.c

```diff
@@ -410,6 +410,7 @@ static int oqs_sig_verify(void *vpoqs_sigctx, const unsigned char *sig, size_t s
     if (ctx_verify) {
       EVP_PKEY_CTX_free(ctx_verify);
     }
+    OQS_SIG_PRINTF2("OQS SIG provider: verify rv = %d\n", rv);
```

Comment:
Is this printf for your debugging and needs to be removed? Or is intended to remain?

Reply:
This is intended to remain alongside all other Debug- and env var-guarded debug statements: It nicely confirms (proper) operation of signature operations as and when OpenSSL triggers them.